ondewo/nlu/agent.proto

```diff
@@ -706,3 +709,38 @@ message ExportResourcesResponse {
     string language_code = 4;
     bytes resource_file = 5;
 }
+message GetModelStatusesRequest {
+    // Required. The project that the agent to fetch is associated with.
+    // Format: `projects/<Project ID>`.
+    string parent = 1;
+    int32 cache_version = 2;
+    string language_code = 3;
+    string model_name = 4;
+}
+
+message ModelStatusProto {
```

Comment:
I would remove "Proto" from the name, since it's obviously a proto message. I know it creates conflicts in names, but it can still be imported as `agent_pb2.ModelStatus` or the `ModelStatus` ORMcan be renamed to `ModelEntry` or `ModelStatusEntry`

Reply:
I think it's still a good way to separate different objects. Yes, here it's obvious that this is a proto message and look redundant, but inside of cai it's not so obvious, when we have two objects with similar names. It's straightforward, but it's also immediately clear, when you read the code what is the purpose of the object.

Reply:
I agree with @ficinator here.

Having a convention to avoid name collisions is totally fine. But, we do not have this convention in the Protos, but in the DB Models.

Some DB Models are called ...Entry which makes sense and we can change them at any point without breaking retro-compatibility.

However, in the protos it is messed up. We cannot clean it easily because we break compatibility and is not going to be ever consistent.